src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -11,13 +17,34 @@
  * про відсутність файлу.</p>
  *
  * <p>Приклад 1: ur-BvT?^ ra w; p
- * Результат 1: abprt</p>
+ * Результат 1: abprt                   </p>
  *
  * <p>Приклад 2: A _f*c a?F
- * Результат 2: acf</p>
+ * Результат 2: acf                      </p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public static final int MAX_LENGTH_LETTER = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> treeSet = new TreeSet<>();
+        try (FileReader fileReader = new FileReader(fileName)) {
+            int character = fileReader.read();
+            while (character != -1) {
+                if (Character.isLetter(character)) {
+                    treeSet.add((char) Character.toLowerCase(character));
+                }
+                character = fileReader.read();
+            }
+        } catch (FileNotFoundException e) {
+            throw new FileNotFoundException("File not found!" + e);
+        }
```

Comment:
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block